tests/integration_tests_control_files/reduced_order/pod_adaptation_snapshot_generator.sh

```diff
@@ -63,7 +63,7 @@ echo "end"
 
 
 dir=$(pwd)
-/usr/bin/mpirun "-n" "1" "$HOME/Codes/PHiLiP/cmake-build-release/bin/PHiLiP_1D" "-i" "${dir}/${file}"
+/usr/bin/mpirun "-n" "1" "$1/PHiLiP_1D" "-i" "${dir}/${file}"
```

Comment:
In `$1/PHiLiP_...`, is `1` a variable holding the directory path? And where is it defined?

Reply:
In Bash, $1 is the first argument given when calling the script. So in my cmake I have ./pod_adaptation_snapshot_generator.sh ${EXECUTABLE_OUTPUT_PATH}; so $1 refers to ${EXECUTABLE_OUTPUT_PATH}.

Reply:
Ah ok, I had forgotten about this, thanks for explaining that!